.github/workflows/build-and-test.yml

```
@@ -3,37 +3,30 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-name: Build and Run Unit Tests
+name: Build and Test
 on:
-  pull_request:
-    paths:
-      - 'src/**'
   push:
-    paths:
-      - 'src/**'
-    branches:
-      - main
+    branches: [ main ]
+  pull_request:
```

Adding `workflow_dispatch` will also allow manual triggering, which is very convenient.

```suggestion
  pull_request
  workflow_dispatch
```